src/indexClinicalData/transformToEsDonor.ts

```diff
@@ -57,9 +53,13 @@ export default (
     createdAt: new Date(mongoDoc.createdAt),
   };
 
-  if (existingEsData) {
-    return { ...defaultRDPCInfo, ...existingEsData, ...clinicalData };
-  } else {
-    return { ...defaultRDPCInfo, ...clinicalData };
+  if (mongoDoc.completionStats?.coreCompletionDate) {
+    clinicalData.coreCompletionDate = new Date(mongoDoc.completionStats.coreCompletionDate);
```

Comment:
this is how the `xFirstPublishedDate` fields are conditionally added to the data object as well